### PR TITLE
Force Full Tune for MT5, Working Story Save for MT5DX+

### PR DESCRIPTION
**Pull request by damon-murdoch**
Force Full Tune for MT5, requires XML option to be added same as MT5DX+/ MT6
Working story save currently tested on first 10-15 stories, allows for stories to be skipped and will still save properly. Win streaks are also preserved, no known bugs yet.

### CODE REVIEW

**Comment by Poliwrath** [comment]
what is commit-and-copy.ps1

**Comment by EmiMidnight** [comment]
holy shit that commit history

File: OpenParrot/src/Functions/Games/ES3X/WMMT5.cpp [hunk 1]
```diff
@@ -425,79 +429,72 @@ unsigned int WINAPI Hook_bind(SOCKET s, const sockaddr *addr, int namelen) {
...
+// Function which runs in a secondary thread if the forceFullTune
+// option is selected in the compiler. If the player's car is not fully
+// tuned, it is forcibly set to max tune. If the player's car is already
+// fully tuned, it is left alone. 
+static DWORD WINAPI forceFullTune(void* pArguments)
+{
+	// Loops while the program is running
+	while (true) {
+
+		// Only runs every 16th frame
```
Comment (derole1) [inline OpenParrot/src/Functions/Games/ES3X/WMMT5.cpp:443]: Just thought I should point out here, this is not every 16 frames, Sleep function works in milliseconds, so you're running your fulltune patch ~62 times per second. I suspect you could probably get away with a much higher value than this to reduce CPU usage a little bit.

Also in quite a few of your comments you refer to bytes as "nibbles". A nibble is 1/2 a byte or 4 bits, ftell and other functions use bytes, which are 8 bits

**Comment by damon-murdoch** [comment]
My apologies - I had no idea it would do that for you, I really thought it would just be a single thing because I was working on a test branch and then rolling them all up in a merge. 

I'll create a new repository with the code in a single commit, and I won't use that same script to commit again :)

File: OpenParrot/src/Functions/Games/ES3X/WMMT5.cpp (hunk 1 above)
Comment (damon-murdoch) [reply OpenParrot/src/Functions/Games/ES3X/WMMT5.cpp:474]: My bad - I'll correct those comments, I appreciate you letting me know :) I'm still getting back into C++ so there are a few things I don't fully understand yet!

**Comment by damon-murdoch** [comment]
> what is commit-and-copy.ps1

Sorry, I thought I removed that from the commit ☠☠ was using it to simplify my workflow lmao

**Comment by Poliwrath** [comment]
Thanks, though I suppose part of the blame is on me for setting github to auto-subscribe to everything for this repo. :)
The script itself is fine, just shouldn't push those commits

**Pull request closed without merging**